src/main/java/seedu/address/storage/JsonSerializableAddressBook.java

```diff
@@ -83,6 +83,8 @@ public AddressBook toModelType() throws IllegalValueException {
             addressBook.addOrder(order);
         }
 
+        addressBook.checkAddressBook();
```

Comment:
The two ways when handling invalid data file:
1. Data file not in correct format -> warning -> start with an empty address book.
2. Data file not in correct format -> IllegalArgumentException -> app does not start.

Should we standardize this to one method??

Reply:
I think would be good if it is the 1st case (less bugs when they smoke test)

Reply:
kk